core/src/main/java/com/klaytn/caver/crypto/KlayCredentials.java

```diff
@@ -24,20 +24,60 @@
 import org.web3j.crypto.Keys;
 import org.web3j.utils.Numeric;
 
+import java.util.*;
+
 import static com.klaytn.caver.wallet.KlayWalletUtils.*;
 
 public class KlayCredentials {
-
-    private final ECKeyPair ecKeyPair;
+    private final List<ECKeyPair> ecKeyPairForTransactionList;
+    private final List<ECKeyPair> ecKeyPairForUpdateList;
+    private final List<ECKeyPair> ecKeyPairForFeeFeePayerList;
     private final String address;
 
     private KlayCredentials(ECKeyPair ecKeyPair, String address) {
-        this.ecKeyPair = ecKeyPair;
+        this.ecKeyPairForTransactionList = Collections.unmodifiableList(Arrays.asList(ecKeyPair));
+        this.ecKeyPairForUpdateList = null;
+        this.ecKeyPairForFeeFeePayerList = null;
+        this.address = address;
+    }
+
+    private KlayCredentials(List<ECKeyPair> ecKeyPairForTransaction, List<ECKeyPair> ecKeyPairForUpdate, List<ECKeyPair> ecKeyPairForFee, String address) {
+        this.ecKeyPairForTransactionList = (ecKeyPairForTransactionArray != null && ecKeyPairForTransactionArray.size() != 0) ? Collections.unmodifiableList(ecKeyPairForTransactionArray) : null;
+        this.ecKeyPairForUpdateList = (ecKeyPairForUpdateArray != null && ecKeyPairForUpdateArray.size() != 0) ? Collections.unmodifiableList(ecKeyPairForUpdateArray) : null;
+        this.ecKeyPairForFeeFeePayerList = (ecKeyPairForFeeArray != null && ecKeyPairForFeeArray.size() != 0) ? Collections.unmodifiableList(ecKeyPairForFeeArray) : null;
         this.address = address;
     }
 
     public ECKeyPair getEcKeyPair() {
-        return ecKeyPair;
+        if (ecKeyPairForTransactionList != null && ecKeyPairForTransactionList.size() > 0) {
```

Comment:
How about extract this code `[List] != null && [List].size() > 0` to method? There are many duplications!

Reply:
I'm going to delete the conditional statement that compares null. Should I implement it as a method?

Reply:
If you do not use this condition, I think you do not need to extract them.